docs/content/utilities/batch.mdx

```diff
@@ -0,0 +1,251 @@
+    @Override
+    public List<String> handleRequest(SQSEvent input, Context context) {
+        List<String> returnValues = PowertoolsSqs.batchProcessor(input, SampleMessageHandler.class); // highlight-line
+
+        return returnValues;
+    }
+
+    public class SampleMessageHandler implements SqsMessageHandler<String> {
+
+        @Override
+        public String process(SQSMessage message) {
+            // This will be called for each individual message from a batch
+            // It should raise an exception if the message was not processed successfully
+            String returnVal = doSomething(message.getBody());
+            return returnVal;
+        }
+    }
+}
+```
+
+You can also use the utility in a more functional way` by providing inline implementation of functional interface <code>SqsMessageHandler#process()</code>
+
+```java:title=App.java
+public class AppSqsEvent implements RequestHandler<SQSEvent, List<String>> {
+
+    @Override
+    public List<String> handleRequest(SQSEvent input, Context context) {
+        // highlight-start
+        List<String> returnValues = PowertoolsSqs.batchProcessor(input, (message) -> {
+            // This will be called for each individual message from a batch
+            // It should raise an exception if the message was not processed successfully
+            String returnVal = doSomething(message.getBody());
+            return returnVal;
+        });
+        // highlight-end
+
+        return returnValues;
+    }
+}
+```
+
+## Passing custom SqsClient
+
+If you need to pass custom SqsClient such as region to the SDK, you can pass your own `SqsClient` to be used by utility either for
+**[SqsBatchProcessor annotation](#SqsBatchProcessor annotation)**, or **[PowertoolsSqs Utility API](#PowertoolsSqs Utility API)**.
+
+```java:title=App.java
+
+public class AppSqsEvent implements RequestHandler<SQSEvent, List<String>> {
+    // highlight-start
+    static {
+        PowertoolsSqs.defaultSqsClient(SqsClient.builder()
+                .build());
+    }
+    // highlight-end
+
+    @Override
+    public List<String> handleRequest(SQSEvent input, Context context) {
+        List<String> returnValues = PowertoolsSqs.batchProcessor(input, SampleMessageHandler.class);
+
+        return returnValues;
+    }
+
+    public class SampleMessageHandler implements SqsMessageHandler<String> {
+
+        @Override
+        public String process(SQSMessage message) {
+            // This will be called for each individual message from a batch
+            // It should raise an exception if the message was not processed successfully
+            String returnVal = doSomething(message.getBody());
+            return returnVal;
+        }
+    }
+}
+
+```
+
+## Suppressing exceptions
+
+If you want to disable the default behavior where `SQSBatchProcessingException` is raised if there are any exception, you can pass the `suppressException` boolean argument.
```

Comment:
What case would this be for?

Reply:
Like if you dont care about if some message fails and still want your function to succeed and then take the heavy lifting yourself.